packages/dd-trace/src/appsec/rasp.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const { storage } = require('../../../datadog-core')
+const addresses = require('./addresses')
+const { httpClientRequestStart } = require('./channels')
+const waf = require('./waf')
+
+function enable () {
+  httpClientRequestStart.subscribe(analyzeSsrf)
```

Comment:
where is this published ?

Reply:
nvm i searched for the name of the variable instead of the name of the channel :facepalm: